### ISSUE TITLE
Use pretrained model with deeplab with different number of output classes

### ISSUE THREAD

**Issue by kartikwar** [tensorflow/models#7376]
What is the top-level directory of the model you are using: deeplab
Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 16.04
TensorFlow installed from (source or binary): pip
TensorFlow version (use command below): tensorflow-gpu 1.13.1
Bazel version (if compiling from source):
CUDA/cuDNN version: CUDA 10.1
GPU model and memory: GeForce RTX 2080

I want to use a pretrained model, every layer except the last layer which outputs labels. Cause in my case the number of labels are not same as the pretrained model . Please help !!!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by sumukhaithal6** [comment]
http://hellodfan.com/2018/07/06/DeepLabv3-with-own-dataset/
